### DISCUSSION TITLE
第3节课笔记(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#233] [笔记]
在这里提交链接

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135532829

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture3-3fde6d75999b497a891e6988354afa17

**Comment by miku8miku** [comment]
https://note.youdao.com/s/1Zi7ut9C

**Comment by hyanworkspace** [comment]
https://github.com/hyanworkspace/InternLM/blob/main/notes/lecture3.md

**Comment by lyyilin** [comment]
https://blog.csdn.net/qq_73668945/article/details/135537921

**Comment by aqizhoua** [comment]
https://aqizhoua.github.io/poster/dd86f549.html

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by Lynn401** [comment]
https://daisy-401.notion.site/Tutorial-3-InternLM-LangChain-7c47803c9e344b6ebc49599feb3ab5be?pvs=4

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_3/note_3.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_3.md

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes/blob/main/lecture3.md

**Comment by TAICHIKF** [comment]
https://github.com/TAICHIKF/InternLM_notes/blob/main/3_Notes_3.pdf

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/5e0370ba-324c-4eac-bdc5-c6287f653ac0

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/blob/main/README.md

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture3.md

**Comment by cyq2368** [comment]
https://mwsnr0a94gx.feishu.cn/docx/NUzAdHquIoApslxI1mOcLSiunQh

**Comment by vjsmmk** [comment]
https://github.com/vjsmmk/puyu/blob/main/3/note/note.pdf

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes/blob/main/lecture3.md

**Comment by leonfrank** [comment]
https://github.com/leonfrank/InternLM-HW/blob/main/lecture3/notes3.pdf

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/lecture/#l3-internlm-langchain

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/fda3o8qrc2xwua9p?singleDoc#
